client/packages/common/src/ui/components/inputs/TextInput/numeric/NumericTextInput.tsx

```diff
@@ -1,20 +1,25 @@
 import React, { FC } from 'react';
 import { StandardTextFieldProps } from '@mui/material';
 import { BasicTextInput } from '../BasicTextInput';
-
-export interface NumericTextInputProps extends StandardTextFieldProps {
+export interface NumericTextInputProps
+  extends Omit<StandardTextFieldProps, 'onChange'> {
+  onChange?: (value: number) => void;
```

Comment:
Is there a use case where onChange is undefined ?

Reply:
Only in the stories atm - it was mainly for consistency with the MUI component which has the property as optional.